docs/graphql/manual/api-reference/pgdump.rst

```diff
@@ -54,8 +54,6 @@ Request
   -  Postgres triggers created by Hasura for event triggers.
 
 
-Source code for the script that is executed can be found `here <https://github.com/hasura/graphql-engine/tree/master/server/src-rsr/run_pg_dump.sh>`_.
```

Comment:
@shahidhk I've seen that you wrote this docs page. I tried to find this file in our repo but can't find it. Can you confirm that we can remove this?